### ISSUE TITLE
JPype deprecation warning message

### ISSUE THREAD

**Issue by tzipperle** [iiasa/ixmp#162]
I have updated message-ix to 1.2.0 via conda and I get the following warning message:

```
-------------------------------------------------------------------------------
Deprecated: convertStrings was not specified when starting the JVM. The default
behavior in JPype will be False starting in JPype 0.8. The recommended setting
for new code is convertStrings=False.  The legacy value of True was assumed for
this session. If you are a user of an application that reported this warning,
please file a ticket with the developer.
-------------------------------------------------------------------------------
```

**Comment by danielhuppmann** [comment]
thanks @tzipperle - at what point do you get the error? During install from conda or when you launch a new `ixmp.Platform` instance?

**Comment by tzipperle** [comment]
When I launch the platform

**Comment by danielhuppmann** [comment]
Do you encounter any other errors? In particular, do the unit tests pass?

**Comment by tzipperle** [comment]
The tests for my model on TravisCI take about an hour. I will let you know after lunch if all the tests are passed.

**Comment by tzipperle** [comment]
All my test have passed. 

The message appears after execute the code line 39 of ixmp/core.py `jpype.startJVM(jpype.getDefaultJVMPath(), *args)`.

I have a small suggestion for the message-ix release notes. A section `Backward Incompatible Changes` would help a bit to classify the changes in `#131: Update clone function argument scen to scenario with planned deprecation of the former.`

**Comment by khaeru** [comment]
@danielhuppmann @zikolach we should:
- decide what the proper value of that JPype argument is,
- specify it, and
- eventually (when JPype < 0.8 is no longer supported) remove the explicit value.

**Comment by danielhuppmann** [comment]
> I have figured out a new problem for me.

Can you please start a new issue for this? The `cplex.opt` file is now created on the fly and removed afterwards, it could be that running several processes in parallel causes this problem.
